monitoring/src/main/java/org/jlab/clas12/monitoring/tof_monitor.java

```diff
@@ -156,6 +159,31 @@ public tof_monitor(int reqrunNum, boolean reqTimeBased, boolean reqwrite_volatil
 		DC_time_odd = new H1F[6][6];
 		DC_time_rescut = new H1F[6][6];
 		DC_time_nocut = new H1F[6][6];
+		//DC jitter histograms, distributions of hits for jitter 0, 1 and 2 and for even and odd event timestamps for each sector/Superlayer
+		DC_jitterzero_sec_sl = new H2F("DC_jitterzero_sec_sl","DC_jitterzero_sec_sl",6,0.5,6.5,6,0.5,6.5);
+		DC_jitterzero_sec_sl.setTitle("jitter 0 hits per sector and sl");
+		DC_jitterzero_sec_sl.setTitleX("sector");
+		DC_jitterzero_sec_sl.setTitleY("superlayer");
+		DC_jitterone_sec_sl = new H2F("DC_jitterone_sec_sl","DC_jitterone_sec_sl",6,0.5,6.5,6,0.5,6.5);
+		DC_jitterone_sec_sl.setTitle("jitter 1 hits per sector and sl");
+		DC_jitterone_sec_sl.setTitleX("sector");
+		DC_jitterone_sec_sl.setTitleY("superlayer");
+		DC_jittertwo_sec_sl = new H2F("DC_jittertwo_sec_sl","DC_jittertwo_sec_sl",6,0.5,6.5,6,0.5,6.5);
+		DC_jittertwo_sec_sl.setTitle("jitter 2 hits per sector and sl");
+		DC_jittertwo_sec_sl.setTitleX("sector");
+		DC_jittertwo_sec_sl.setTitleY("superlayer");
+		DC_hits_even_ts_sec_sl = new H2F("DC_hits_even_ts_sec_sl","DC_hits_even_ts_sec_sl",6,0.5,6.5,6,0.5,6.5);
+		DC_hits_even_ts_sec_sl.setTitle("hits per sector and sl for even time stamps");
+		DC_hits_even_ts_sec_sl.setTitleX("sector");
+		DC_hits_even_ts_sec_sl.setTitleY("superlayer");
+		DC_hits_odd_ts_sec_sl = new H2F("DC_hits_odd_ts_sec_sl","DC_hits_odd_ts_sec_sl",6,0.5,6.5,6,0.5,6.5);
+		DC_hits_odd_ts_sec_sl.setTitle("hits per sector and sl for odd time stamps");
+		DC_hits_odd_ts_sec_sl.setTitleX("sector");
+		DC_hits_odd_ts_sec_sl.setTitleY("superlayer");
```

Comment:
Then with `Enum` you can do something like this (untested):
```suggestion
		for(DCenum e : DCenum.values()) {
		  DC_hits.put(e, new H2F("DC_"+e.name()+"_sec_sl", "DC_"+e.name()+"_sec_sl", 6, 0.5, 6.5, 6, 0.5, 6.5));
		  DC_hits.get(e).setTitleX("sector");
		  DC_hits.get(e).setTitleY("superlayer");
		}
		DC_hits.get(jitterzero).setTitle("jitter 0 hits per sector and sl");
		DC_hits.get(jitterone).setTitle("jitter 1 hits per sector and sl");
		DC_hits.get(jittertwo).setTitle("jitter 2 hits per sector and sl");
		DC_hits.get(hits_even_ts).setTitle("hits per sector and sl for even time stamps");
		DC_hits.get(hits_odd_ts).setTitle("hits per sector and sl for odd time stamps");
```

Reply:
Yes we could do that in the next iteration. Thanks for the suggestion